lib/puppet/file_serving/configuration.rb

```diff
@@ -80,15 +80,10 @@ def umount(name)
 
   def mk_default_mounts
     @mounts["modules"] ||= Mount::Modules.new("modules")
-    @mounts["modules"].allow('*') if @mounts["modules"].empty?
```

Comment:
What were these for?

Reply:
Related to above, previously the hard coded mounts had to be global allowed, on the off chance that the legacy auth was enabled. Now that puppetserver's auth.conf performs the check, we no longer need to store allow/deny rules for mounts.